src/utils.js

```diff
@@ -74,20 +74,34 @@ export const exportToDisk = function exportToDisk(
     }
 };
 
+/**
+ * @summary Converts position array of Buffer Geometry to vertices vectors
+ * @param positionArray
+ */
+function convertPositionToVertices(geometry, position) {
```

Comment:
1. Add param for geometry
2. positionArray -> position

Reply:
Done